worker/backend/backend.go

```diff
@@ -106,7 +112,15 @@ func (b *Backend) Create(gdnSpec garden.ContainerSpec) (container garden.Contain
 //
 // Errors:
 // * TODO.
-func (b *Backend) Destroy(handle string) (err error) { return }
+func (b *Backend) Destroy(handle string) (err error) {
+	var ctx = namespaces.WithNamespace(context.Background(), b.namespace)
+
+	if handle == "" {
+		return InputValidationError{}
```

Comment:
it might be useful to have a string passed to this, so that we can add some more context to the error here 🤔  

e.g.:

```go
InputValidationError{"missing handle"}
```

so that we can not only check the error type, but also reason about the low-level detail that caused the validation to fail

Reply:
support

Reply:
We'll make a note to do this for all the error types when we're a little further along!